wagtail/images/tests/tests.py

```diff
@@ -237,7 +238,7 @@ def test_rich_text_with_missing_image(self):
 class TestFormat(WagtailTestUtils, TestCase):
     def setUp(self):
         # test format
-        self.format = Format("test name", "test label", "test classnames", "original")
+        self.format = Format("test name", "test label", "test is-primary", "original")
```

Comment:
Just updated this test code to be a bit more real world and avoid the confusing naming of a class of classnames.